libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/Notification.java

```diff
@@ -20,27 +20,24 @@
 
 public class Notification {
 
-    // Denotes notification type. MEMBERSHIP_APPROVAL, MEMBERSHIP_EXPIRY etc.
```

Comment:
Removed the type from notification as it is no longer required (was used in EmailNotificationService in a switch statement to determine which body / subject to generate.